tests/aiohttp/test_aiohttp_api_secure.py

```diff
@@ -159,3 +159,30 @@ async def test_async_secure(aiohttp_api_spec_dir, aiohttp_client):
     response = await app_client.get('/v1.0/async_bearer_auth', headers={'Authorization': bearer_header})
     assert response.status == 200
     assert (await response.json()) == {"scope": ['myscope'], "uid": 'test-user'}
+
+
+async def test_auth_exception_swagger(oauth_aiohttp_client, aiohttp_api_spec_dir, aiohttp_client):
```

Comment:
Can't we parametrize the swagger / openapi?

Reply:
Yes, we can 🙂